lib/controller.js

```diff
@@ -7,7 +7,10 @@ var sprintf = require('sprintf-js').sprintf;
 var cp = require('child_process');
 var async = require('async');
 var updates = require('./update-fetch');
+var debug = require('debug')('controller');
+var Menu = require('terminal-menu');
 var controller = {};
+controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH + '/id_rsa';
```

Comment:
- [Tessel.TESSEL_AUTH_PATH + '/id_rsa'](https://github.com/Student007/t2-cli/blob/master/lib/lan_connection.js#L32) -> used in lan_connection.js
- [path.join(os.tmpDir(),'id_rsa')](https://github.com/ericvicenti/ssh-keygen/blob/master/src/ssh-keygen.js#L98) -> extern ssh-keygen module
- [filepath](https://github.com/Student007/t2-cli/blob/master/lib/tessel/provision.js#L3-L13) -> provision.js

`Tessel` is included in `provision.js` and also `lan_connection.js` but useless in `bin/tessel-2.js`!
The only solution would be to use `controller.js` to provide that *const* because `controller.js` is included within `bin/tessel-2.js`. A possible solution could be:
```
controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH+'/id_rsa';
```
within [controller.js](https://github.com/Student007/t2-cli/blob/master/lib/controller.js#L10)

For this reason I decided to create `controller.TESSEL_AUTH_KEY`